src/components/LoginBar/LoginBar.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+type LoginBarProps = {
+  isLogged?: boolean;
+};
+
+const LoginBarMenuStyled = styled.div`
+  padding: 12px 16px;
+  height: 56px;
+  background-color: ${props => props.theme.colors.main.primary};
+  display: flex;
+`;
+
+const LoginButton = styled.div`
+  display: flex;
+  align-items: center;
+  ${props => props.theme.typography.subtitle1};
+  :hover {
+    background-color: rgba(85, 133, 255, 0.2);
+  }
+`;
+
+const StyledLink = styled(Link)`
+  color: ${props => props.theme.colors.utils.background.light};
+  text-decoration: none;
+`;
+export const LoginBar = ({ isLogged }: LoginBarProps) => {
+  return (
+    <>
```

Comment:
hey - did You update this PR with latest changes from master - I wrote comment about this piece of code in previously created PR?